src/constants/label.js

```diff
@@ -175,7 +175,7 @@ export function replaceExpression(
  * Increasing this constant deepens recursion for replacing delimiters in the
  * list, potentially affecting style loading performance.
  */
-const maxValueListLength = 9;
+const maxValueListLength = 3;
```

Comment:
Three values separated by two semicolons is plenty enough for the `name`s currently using semicolons. In the future, with a more efficient string replacement implementation, we can add support for more semicolons, which will make this option more attractive to mappers currently using informal delimiters like spaces.